### PR TITLE
#166790061 user should be able to view notifications that they receive

### PR DESCRIPTION
**Pull request by shaluchandwani**
#### What does this PR do?

Allow user to view notifications in real-time

#### Description of Task to be completed?

Users should be able to view notifications

#### How should this be manually tested?

Login to Authors havens the check your profile picture in the top right side of the home page and a bell icon after that.

After cloning the repo, cd into it, type npm run test. To run the application type npm start

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/166790061

#### Screenshots (if appropriate)

ezgif-4-5645a2435de4
ezgif-4-5846cd0dbb52

### CODE REVIEW

**Review by kagabof — CHANGES_REQUESTED** [review]
Increase the margin on the profile page.

**Review by kagabof — APPROVED** [review]

**Review by pngabo — APPROVED** [review]

File: src/components/Profile/ProfileEdit/ProfileEditPicture/ProfileEditPicture.js [hunk 1]
```diff
@@ -86,7 +85,7 @@ export class ProfileEditPicture extends Component {
               onChange={this.handleChange}
               isRequired={true}
             />
-            <Button type="submit" loading={loading}>
+            <Button type="submit" class="button1" loading={loading}>
```
Comment (pngabo) [inline src/components/Profile/ProfileEdit/ProfileEditPicture/ProfileEditPicture.js]: @shaluchandwani Please replace class with className.

File: src/components/Profile/ProfileEdit/ProfileEdit.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ export class ProfileEdit extends Component {
     const { modalStyle, user } = this.state;
     return (
       <div className="ProfileEdit">
-        <Button type="button" onClick={this.showModal}>
+        <Button type="button" class="button1" onClick={this.showModal}>
```
Comment (pngabo) [inline src/components/Profile/ProfileEdit/ProfileEdit.js]: @shaluchandwani Please replace class with className

File: src/components/Profile/ProfileEdit/ProfileEditForm/ProfileEditForm.js [hunk 3]
```diff
@@ -168,8 +151,7 @@ export class ProfileEditForm extends Component {
             />
           </div>
           <div className="small-screen-4">
-            {/* <Button type="submit" loading={loading} onClick={this.handleOnClick}> */}
-            <Button type="submit" loading={loading} >
+            <Button type="submit" class="button1" loading={loading} >
```
Comment (pngabo) [inline src/components/Profile/ProfileEdit/ProfileEditForm/ProfileEditForm.js]: Please replace class with className

File: src/components/Header/Notifications/getNotifications.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    showModal: false,
+    notification_number: 0,
+  };
+
+  closeModal = (e) => {
+    e.preventDefault();
+    this.setState({ showModal: false });
+  };
+
+  componentDidMount = async () => {
```
Comment (billkabanga) [inline src/components/Header/Notifications/getNotifications.js]: Great work on this PR @shaluchandwani, Avoid making a lifecycle method asynchronous, I would rather you create an asynchronous function to fetch the notifications and invoke it in the lifecycle method.

File: src/components/Profile/ProfileUserDetails/ProfileUserDetails.js [hunk 5]
```diff
@@ -35,18 +35,23 @@ export class ProfileUserDetails extends Component {
           firstName, lastName, username, email, bio, image,
         },
       } = this.props;
+      let image1;
+      if (image) {
+        // eslint-disable-next-line no-unused-expressions
+        image.split(':')[0] === 'https' ? image1 = image : image1 = `https://res.cloudinary.com/djxhcwowp/image/upload/v${image}`;
```
Comment (billkabanga) [inline src/components/Profile/ProfileUserDetails/ProfileUserDetails.js:41]: Why did you disable linting for this line? It looks like there's a variable that is unused.

Comment (shaluchandwani) [reply src/components/Profile/ProfileUserDetails/ProfileUserDetails.js:41]: No, the issue was that whole line was receiving an error. All the variables are used in that. that is why I put this.

**Pull request merged**